subsys/zigbee/osif/zb_nrf_crypto.c

```diff
@@ -105,7 +105,7 @@ zb_uint32_t zb_random_seed(void)
 void zb_osif_aes_init(void)
 {
 #if CONFIG_CRYPTO_NRF_ECB
-	dev = device_get_binding(CONFIG_CRYPTO_NRF_ECB_DRV_NAME);
```

Comment:
Does that mean we can remove CONFIG_CRYPTO_NRF_ECB_DRV_NAME too?

Reply:
It came from upstream and is already gone (d855d472889cd32861655dff20fa2a3546ffe416)